integration-tests/src/test/groovy/com/okta/sdk/tests/it/UserTypesIT.groovy

```diff
@@ -15,7 +15,7 @@
  */
 package com.okta.sdk.tests.it
 
-import com.okta.sdk.resource.UserType
+import com.okta.sdk.resource.user.type.UserType
```

Comment:
Are there other Objects (or will there be) in the `user.type` package?

Reply:
Presently nothing else, not sure what future holds :)